packages/trader/src/AppV2/Utils/trade-types-utils.tsx

```diff
@@ -0,0 +1,232 @@
+            </Text>
+        </React.Fragment>
+    ),
+    [TERM.TAKE_PROFIT]: (
+        <Localize i18n_default_text='If you select this feature, your trade will be closed automatically at the nearest available asset price when your profit reaches or exceeds the take profit amount. Your profit may be more than the amount you entered depending on the market price at closing.' />
+    ),
+};
+
+export const getTermDefinition = ({ contract_type, term }: { contract_type: string; term: string }) => {
+    const result = DEFINITION[term];
+    if (typeof result === 'function') return result(contract_type);
+    return result ?? '';
+};
+
+export const getContractDescription = (
+    content: {
+        type: string;
+        text: JSX.Element | string;
+    }[]
+) =>
+    content.map(({ type, text }, index) => {
+        if (type === 'heading' && typeof text !== 'string')
+            return (
+                <Text key={text.props.i18n_default_text} bold size='md' className='description__heading'>
+                    {text}
+                </Text>
+            );
+        if ((type === 'paragraph' || type === 'general') && typeof text !== 'string')
+            return (
+                <Text
+                    as='p'
+                    key={text.props.i18n_default_text}
+                    size='sm'
+                    className={`description__${type}`}
+                    color='quill-typography__color--prominent'
+                >
+                    {text}
+                </Text>
+            );
+        if (type === 'video' && typeof text === 'string')
+            return <VideoFragment contract_type={text} key={text + index} />;
+    });
+
+export const DESCRIPTION_VIDEO_IDS: TDtraderVideoUrl = {
+    [CONTRACT_LIST.ACCUMULATORS]: {
+        light: DESCRIPTION_VIDEO_ID.accumulator.light,
+        dark: DESCRIPTION_VIDEO_ID.accumulator.dark,
+    },
+    [CONTRACT_LIST.EVEN_ODD]: {
+        light: DESCRIPTION_VIDEO_ID.even_odd.light,
+        dark: DESCRIPTION_VIDEO_ID.even_odd.dark,
+    },
+    [CONTRACT_LIST.HIGHER_LOWER]: {
+        light: DESCRIPTION_VIDEO_ID.high_low.light,
+        dark: DESCRIPTION_VIDEO_ID.high_low.dark,
+    },
+    [CONTRACT_LIST.MATCHES_DIFFERS]: {
+        light: DESCRIPTION_VIDEO_ID.match_diff.light,
+        dark: DESCRIPTION_VIDEO_ID.match_diff.dark,
+    },
+    [CONTRACT_LIST.MULTIPLIERS]: {
+        light: DESCRIPTION_VIDEO_ID.multiplier.light,
+        dark: DESCRIPTION_VIDEO_ID.multiplier.dark,
+    },
+    [CONTRACT_LIST.OVER_UNDER]: {
+        light: DESCRIPTION_VIDEO_ID.over_under.light,
+        dark: DESCRIPTION_VIDEO_ID.over_under.dark,
+    },
+    [CONTRACT_LIST.RISE_FALL]: {
+        light: DESCRIPTION_VIDEO_ID.rise_fall.light,
+        dark: DESCRIPTION_VIDEO_ID.rise_fall.dark,
+    },
+    [CONTRACT_LIST.TOUCH_NO_TOUCH]: {
+        light: DESCRIPTION_VIDEO_ID.touch.light,
+        dark: DESCRIPTION_VIDEO_ID.touch.dark,
+    },
```

Comment:
```suggestion
    [CONTRACT_LIST.TOUCH_NO_TOUCH]: DESCRIPTION_VIDEO_ID.touch,
```
can we try a shorter version ?🙏🏻 :)

Reply:
Of course! Thanks for noticing